treeherder/webapp/api/runnable_jobs.py

```diff
@@ -14,6 +17,21 @@ def list(self, request, project):
         """
         GET method implementation for list of all runnable buildbot jobs
         """
+        if 'decisionTaskID' in request.query_params and len(request.query_params['decisionTaskID']):
+            decisionTaskID = request.query_params['decisionTaskID']
+            tc_jobs_url = "https://queue.taskcluster.net/v1/task/" + decisionTaskID + "/artifacts/public/full-task-graph.json"
+            tc_graph = None
+            validate = URLValidator()
+            try:
+                validate(tc_jobs_url)
+                tc_graph = fetch_json(tc_jobs_url)
+            except ValidationError:
```

Comment:
I'm slightly confused why we `pass` on validation exceptions? Could you add a comment?

Also, we should catch the HTTPErrors that could occur from `fetch_json` and set appropriate Response codes (HTTP 500 likely isn't appropriate for all).